podget

```diff
@@ -1142,7 +1142,8 @@ PLAYLIST_ConvertToASX() {
 
      # Removing unix2dos dependency. Converting to sed statement with in-place editing of the file in question.
      # ctrl-v ctrl-m for windows line end.
-     sed -i 's/$/
/' "${DIR_LIBRARY}"/"${ASX_PLAYLISTNAME}"
+     sed -i 's/$/
+/' "${DIR_LIBRARY}"/"${ASX_PLAYLISTNAME}"
```

Comment:
Is this change needed?  It's been a little while since I tested dos file to unix conversions but as I recall the ctrl-M was the way to go....